tests/downloader/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
you could squash  with downloader implementation commit

Reply:
Done